youthful-piety/original/09-chapter-08.adoc

```diff
@@ -352,7 +361,9 @@ beyond all doubt, that all those who trust in Him, will not lack the bread of li
 The following extracts from a diary which he kept,
```

Comment:
You forgot to close the embedded paragraph here with a -- (but only here in the original. The modernized is fine). 👍🏻

Reply:
It's crazy I miss these when I look over the PDF's because I **specifically** compare the embedded things to make sure they match....because when they don't, they whole rest of the document is wrong too, so idk how I can miss them. idk what to say! :)